src/status_im/ui/screens/events.cljs

```diff
@@ -224,8 +238,7 @@
         "local_storage.set"       (dispatch [:set-local-storage event])
```

Comment:
Why is this `local_storage` signal handled in two places, here and in `case` statement of the `handler-jail-signal` function ? Or are those two different events ?

Reply:
that's leftover, removed, thanks